src/index.css

```diff
@@ -314,14 +314,14 @@ Notes on possible css class implementation:
 	}
 }
 
-.dropup .dropdown-menu {
+.dropup > .dropdown-menu {
```

Comment:
Be aware that these styles are used in elementsui docs only. The corresponding changes should be made in Elements dropdown.less